front/components/vaults/VaultSideBarMenu.tsx

```diff
@@ -111,16 +111,18 @@ export default function VaultSideBarMenu({
                     label={sectionLabel}
                     variant="secondary"
                   />
-                  {sectionLabel === "PRIVATE" && isAdmin && (
-                    <Button
-                      className="mt-4"
-                      size="xs"
-                      variant="tertiary"
-                      label="Create Vault"
-                      icon={LockIcon}
-                      onClick={() => setShowVaultCreationModal(true)}
-                    />
-                  )}
+                  {sectionLabel === "PRIVATE" &&
```

Comment:
we should show the button but have a dialog that say they hit the limit when clicked. We already have reusable components for that I believe.

Reply:
ok 👍

Reply:
Should we display the private section and "Add vaults" button in all cases, even if the limit is 0 ? Or hide everything as if feature was not enabled ?

Reply:
We should not have limit 0 in general right? So for now I think we can just show when the flag is enabled independent of limit and have a popup if we hit the limit.

If tomorrow we have plans with 0 vaults it might make sense to hide the button but I don't think we need to solve that right now

Reply:
there's no plan with "0" vaults right now, so yes we can ignore that and leave the button in all cases for now